src/main.rs

```diff
@@ -46,9 +41,17 @@ pub fn main() {
     if expected_version.is_some() {
         assert_eq!(version.parsed, version.parsed);
```

Comment:
We should probably make his something like: `assert_eq!(version.parsed, expected_version)` or such, but that can be a separate PR.